### PR TITLE
Use ansible instead of awscli

### PR DESCRIPTION
**Pull request by jstodola**
--- Description of proposed changes ---

The following parts of the script haven't been rewritten to ansible yet:
* Import of the image from S3 as a snapshot
* Registerring AMI image from the imported snapshot.

Ansible modules do not support importing an s3 object as a snapshot, reported as:
https://github.com/ansible/ansible/issues/53453

A possible workaround using the image_location parameter of the ec2_ami ansible module is being investigated.

--- Merge policy ---

- [ ] Travis CI PASS
- [x] `*-aws-runtest` PASS
- [x] `*-azure-runtest` PASS
- [ ] `*-images-runtest` PASS
- [x] `*-openstack-runtest` PASS
- [x] `*-vmware-runtest` PASS
- [ ] For `rhel8-*` and `rhel7-*` branches commit log references an approved
  bug in Bugzilla. Do not merge if the bug doesn't have the 3 ACKs set to `+`!

--- Jenkins commands ---

- `ok to test` to accept this pull request for testing
- `test this please` for a one time test run
- `retest this please` to start a new build

### CODE REVIEW

**Comment by larskarlitski** [comment]
Nice! Looks like this is much less complex than using `awscli` directly. Is there another benefit of switching to ansible?

Is that role using `awscli` under the hood? Is it ok that we're not testing with `awscli` anymore?

Is the idea to switch tests for other cloud providers to ansible as well?

I think it'd be more readable if the playbook were in separate files.

File: tests/cli/test_build_and_deploy_aws.sh [hunk 1]
```diff
@@ -38,7 +38,7 @@ rlJournalStart
             rlAssertRpm python3-pip
         fi
 
-        rlRun -t -c "pip3 install awscli"
+        rlRun -t -c "pip3 install awscli ansible[aws]"
```
Comment (atodorov) [inline tests/cli/test_build_and_deploy_aws.sh]:
```
STDOUT: TASK [Make sure bucket exists] ***********************************************************************************************************
STDOUT: fatal: [localhost]: FAILED! => {"changed": false, "msg": "Python modules \"botocore\" or \"boto3\" are missing, please install both"}
```

Comment (atodorov) [reply tests/cli/test_build_and_deploy_aws.sh]: Installing python3-boto3 from RPM fixed it for me. (Or if you prefer from pypi).

**Comment by atodorov** [comment]
> I think it'd be more readable if the playbook were in separate files.

@larskarlitski how do we pass specific values then (which we have as variables in the script) ?

**Comment by jstodola** [comment]
I agree that separating the ansible playbooks makes sense to improve readability. Passing the values can be done via --extra-vars: 
ansible-playbook release.yml --extra-vars "version=1.23.45 other_variable=foo"
Doc: https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#passing-variables-on-the-command-line

File: tests/cli/test_build_and_deploy_aws.sh (hunk 1 above)
Comment (jstodola) [reply tests/cli/test_build_and_deploy_aws.sh]: Fixed, python3-boto3 rpm is now installed

**Comment by jstodola** [comment]
Latest version installs python3-boto3 rpm and ansible playbooks are in separate files.

**Comment by jstodola** [comment]
Removed .instance.yml.swp, thanks for catching it.
The cleanup script will be tracked in a separate PR.

**Pull request merged**